package.json

```diff
@@ -4,7 +4,7 @@
   "description": "JS Code Standards for all SilverMine projects - eslint enforcement",
   "main": "index.js",
   "scripts": {
-    "test": "check-node-version --npm 6.14.12",
+    "test": "check-node-version --npm 8.5.5",
```

Comment:
With this upgrade, should there also be an update to the package-lock.json to update it to the v2 format?